support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -333,11 +342,14 @@ export function ThreeTierLanding(): JSX.Element {
 		);
 	};
 
-	const getCardContentBaseObject = (cardTier: 1 | 2 | 3) => {
+	const getCardContentBaseObject = (
+		cardTier: 1 | 2 | 3,
+		contributionTypeKeyOverride?: 'annual' | 'monthly',
+	) => {
 		const tierPlanCountryCharges =
-			tierCards[`tier${cardTier}`].plans[contributionTypeKey].charges[
-				countryGroupId
-			];
+			tierCards[`tier${cardTier}`].plans[
+				contributionTypeKeyOverride ?? contributionTypeKey
```

Comment:
I get the logic, but it is quite complicated for what I think can just be hardcoded values?

Reply:
Wanted to be on safe-side, there was talk of tweaking prices on meeting earlier and not sure if this Tote offer could overrun into that.